doc/ref/cli/salt-key.rst

```diff
@@ -124,10 +134,20 @@ Actions
 .. option:: -d DELETE, --delete=DELETE
 
     Delete the specified key. Globs are supported.
+    .. warning::
+        Using globs to reject the specified public key may cause performance issues
+        because of a Salt master degredation dramatic performance salt-master degradation. 
+        See [`performance guide`](https://docs.saltproject.io/en/latest/topics/tutorials/intro_scale.html#too-many-minions-re-authing) 
+        for more information about performance impacts and workarounds.
 
 .. option:: -D, --delete-all
 
     Delete all keys.
+    .. warning::
+        Using globs to reject the specified public key may cause performance issues
+        because of a Salt master degredation dramatic performance salt-master degradation. 
+        See [`performance guide`](https://docs.saltproject.io/en/latest/topics/tutorials/intro_scale.html#too-many-minions-re-authing) 
+        for more information about performance impacts and workarounds.
```

Comment:
This warning needs say something more along the lines of "deleting many keys simultaneously may cause performance issues" instead of talking about globbing as there is no globbing here.